library/koji_tag.py

```diff
@@ -40,6 +40,12 @@
      description:
        - dict of package owners and the a lists of packages each owner
          maintains.
+       - If you omit this "packages" setting, Ansible will not edit the
+         existing package list for this tag. For example, if you have another
+         system that already manages the package list for this tag, you may
+         want to omit this setting in Ansible.
+         https://github.com/project-ncl/pnc is one example of a system that
```

Comment:
siege linked me to https://github.com/project-ncl/causeway , which is really the thing that's doing this package list manipulation (via kojiji)